backends/include/dppl_sycl_queue_interface.h

```diff
@@ -63,4 +63,13 @@ DPPLQueue_GetDevice (__dppl_keep const DPPLSyclQueueRef QRef);
 DPPL_API
 void DPPLQueue_Delete (__dppl_take DPPLSyclQueueRef QRef);
 
+/*!
+ * @brief C-API wrapper for sycl::queue::memcpy. It waits an event.
+ *
+ * @param    QRef           A DPPLSyclQueueRef pointer that gets deleted.
```

Comment:
Is this an appropriate comment? Looks like it had been copy-pasted from the declaration above.

Reply:
Fixed.